src/components/Radio/index.tsx

```diff
@@ -1,145 +1,125 @@
-import { Theme, css } from '@emotion/react'
 import styled from '@emotion/styled'
-import { transparentize } from 'polished'
 import PropTypes from 'prop-types'
-import { InputHTMLAttributes, ReactNode, useMemo } from 'react'
+import { InputHTMLAttributes, ReactNode } from 'react'
 import { Radio as ReakitRadio } from 'reakit'
-import Box, { XStyledProps } from '../Box'
-import Expandable from '../Expandable'
 import Icon from '../Icon'
 import Typography from '../Typography'
 
-const StyledIcon = styled(Icon)``
-
-const IconContainer = styled.div`
-  min-width: 32px;
-  width: 32px;
-  height: 32px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  margin-right: 8px;
+const StyledIcon = styled(Icon)`
+  margin-right: ${({ theme }) => theme.space['1']};
+  margin-top: ${({ theme }) => theme.space['0.25']};
+  border-radius: ${({ theme }) => theme.radii.circle};
+  fill: ${({ theme }) => theme.colors.neutral.text};
```

Comment:
```suggestion
  fill: ${({ theme }) => theme.colors.neutral.textWeak};
```

Seen with design we will keep using this token color for checkbox and radio, they will then update the token to make it more accessible

Reply:
done]